src/StatsdClient/Statsd.cs

```diff
@@ -219,9 +220,14 @@ public void Add(string title, string text, string alertType = null, string aggre
         }
 
         public void Send(string title, string text, string alertType = null, string aggregationKey = null, string sourceType = null, int? dateHappened = null, string priority = null, string hostname = null, string[] tags = null, bool truncateIfTooLong = false)
+        {
+            SendAsync(title, text, alertType, aggregationKey, sourceType, dateHappened, priority, hostname, tags, truncateIfTooLong).Wait();
+        }
+
+        public async Task SendAsync(string title, string text, string alertType = null, string aggregationKey = null, string sourceType = null, int? dateHappened = null, string priority = null, string hostname = null, string[] tags = null, bool truncateIfTooLong = false)
         {
             truncateIfTooLong = truncateIfTooLong || TruncateIfTooLong;
-            Send(Event.GetCommand(title, text, alertType, aggregationKey, sourceType, dateHappened, priority, hostname, tags, truncateIfTooLong));
+            await SendAsync(Event.GetCommand(title, text, alertType, aggregationKey, sourceType, dateHappened, priority, hostname, tags, truncateIfTooLong)).ConfigureAwait(false);

```

Comment:
Can't you just return SendAsync instead of `await`-ing here?  They both return a `Task`.  You can also remove async from the method signature.

The caller is responsible for using `await`-ing on this method in their async method.